src/App.tsx

```diff
@@ -34,62 +35,133 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort();
+  }
+
+  if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((a, b) => a.length - b.length);
```

Comment:
You have a limited amount of sortTypes, so you can use switch...case here instead.

Reply:
Done